include/thread_pool/queue_parameters.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <concurrentqueue.h>
+
+namespace tp {
+
+struct QueueTraits : public moodycamel::ConcurrentQueueDefaultTraits {
+#if defined(__rtems__)
```

Comment:
could you use `#ifndef __linux__` instead of` #if defined(__rtems__)`
Also, provide default values for linux case?